Assignment6.Rmd

```diff
@@ -57,6 +57,8 @@ post(c.tree2, file = "tree2.ps", title = "MOOC") #This creates a pdf image of th
 
 #Now use both the original tree and the pruned tree to make predictions about the the students in the second data set. Which tree has a lower error rate?
 
+The two outpout are the same, two trees are the same.
+
```

Comment:
That doesn't sound right. Are you sure the second tree has been pruned? The issue may be the inclusion of forum posts, that makes a reasonably predictive but not very complex tree.